src/Onboarding/FormRepository.php

```diff
@@ -72,25 +74,38 @@ protected function isFormAvailable($formID)
     }
 
     /**
+     * @unreleased Replace "Donation Form" with "Campaign Form"
      * @since 3.15.0 Create the default v3 form.
      * @since 2.8.0
      * @return int Form ID
      *
+     * @throws Exception
      */
-    protected function makeAndPersist()
+    protected function makeAndPersist(): int
     {
-        $form = new DonationForm([
-            'title'    => __('GiveWP Donation Form', 'give'),
-            'status'   => DonationFormStatus::PUBLISHED(),
-            'settings' => FormSettings::fromArray([
-                'designId' => 'multi-step',
-                'designSettingsImageUrl' => GIVE_PLUGIN_URL . '/assets/dist/images/admin/onboarding/header-image.jpg',
-                'designSettingsImageStyle' => 'above',
-                'designSettingsImageAlt' => 'GiveWP Onboarding Donation Form',
-            ]),
-            'blocks'   => (new GenerateDefaultDonationFormBlockCollection())(),
+        $campaign = Campaign::create([
+            'type' => CampaignType::CORE(),
+            'title' => __('GiveWP Donation Form', 'give'),
+            'shortDescription' => '',
+            'longDescription' => '',
+            'logo' => '',
+            'image' => '',
+            'primaryColor' => '#0b72d9',
+            'secondaryColor' => '#27ae60',
+            'goal' => 1000,
+            'goalType' => CampaignGoalType::AMOUNT(),
+            'status' => CampaignStatus::ACTIVE(),
         ]);
 
+        $form = DonationForm::find($campaign->defaultFormId);
```

Comment:
Since models are nullable we should always be checking if it exists before accessing properties to avoid fatal errors

Reply:
Resolved in 144bde14c13545b9d405d5ab215ed906d13a039e